hoomd/md/PotentialPair.h

```diff
@@ -83,7 +83,8 @@
     now, that will be queried from the evaluator.
     \sa export_PotentialPair()
 */
-template<class evaluator> class PotentialPair : public ForceCompute
+template<class evaluator, typename extra_pkg = std::nullptr_t>
+class PotentialPair : public ForceCompute
```

Comment:
These are the minimum changes I felt were possible for the polymorphism/code injection I was attempting to achieve to make maintenance easier for such specialized pair interactions.

Reply:
A template parameter specifying the type of the information that will be created, utilized once per pair, and destroyed.